package.json

```diff
@@ -201,7 +204,7 @@
     "postcss-colorfix": "^0.0.4",
     "postcss-loader": "3.0.0",
     "postcss-nested": "^4.2.1",
-    "prettier": "1.19.1",
+    "prettier": "2.1.2",
```

Comment:
This upgrade is needed since the older version prettier throws error when dealing with `.mdx` files